src/univariate/continuous/uniform.jl

```diff
@@ -101,6 +101,27 @@ function mgf(d::Uniform, t::Real)
     v = (a + b) * t / 2
     exp(v) * (sinh(u) / u)
 end
+function expfd0_taylor(x)
+    # taylor series of (exp(x) - 1) / x
+    T = typeof(x)
+    a0 = inv(T(1))
+    a1 = inv(T(2))
+    a2 = inv(T(6))
+    a3 = inv(T(24))
+    a4 = inv(T(120))
+    @evalpoly(x, a0, a1, a2, a3, a4)
+end
```

Comment:
I am worried that this is not sufficiently accurate for non-standard (i.e. non-`Float64`) number types.

Reply:
Note that we only use this internal function it in the region ` sqrt(eps(float(one(x))))`. So the `x^4` is less than `eps(x)^2`. In fact I think ` @evalpoly(x, a0, a1, a2)` would already reach machine precision in this range. I included the higher order terms only to also get automatic differentiation accurate.

Reply:
Sure, I assumed that you tested it with Float64 and that it is sufficiently accurate there. But it might still be too inaccurate for e.g. higher-order differentiation. Or BigFloat. Or Float128. And for Float16 or Float32 it might be a bit inefficient since possibly one could remove terms. Hence I think usually such series approximations should either be computed dynamically (e.g. until numerical convergence) or be restricted to the specific type that they were designed for.